client/my-sites/domains/domain-management/list/test/index.js

```diff
@@ -117,129 +117,6 @@ describe( 'index', () => {
-				expect( component.state( 'primaryDomainIndex' ) ).toBe( 1 );
-			} );
-
-			describe( '#handleUpdatePrimaryDomain', () => {
-				let setPrimaryDomainStub;
-				let setPrimaryDomainResolve;
-				let setPrimaryDomainReject;
-				beforeEach( () => {
-					setPrimaryDomainStub = jest
-						.spyOn( component.instance(), 'setPrimaryDomain' )
-						.mockReturnValue(
-							new Promise( ( resolve, reject ) => {
-								setPrimaryDomainResolve = resolve;
-								setPrimaryDomainReject = reject;
-							} )
-						);
-				} );
-
-				test( 'should not call setPrimaryDomain with on trying to set the already primary domain', () => {
-					component.instance().handleUpdatePrimaryDomain( 1, defaultProps.domains[ 1 ] );
-					setPrimaryDomainResolve();
-					expect( component.state( 'settingPrimaryDomain' ) ).toBe( false );
-					expect( component.state( 'changePrimaryDomainModeEnabled' ) ).toBe( false );
-					expect( setPrimaryDomainStub ).not.toHaveBeenCalled();
-				} );
-
-				test( 'should call setPrimaryDomain with a domain name', () => {
-					return new Promise( ( done ) => {
-						component.instance().handleUpdatePrimaryDomain( 0, defaultProps.domains[ 0 ] );
-						expect( component.state( 'settingPrimaryDomain' ) ).toBe( true );
-						expect( component.state( 'primaryDomainIndex' ) ).toBe( 0 );
-						// `.setPrimaryDomain` should be called with the domain name
-						expect( setPrimaryDomainStub ).toHaveBeenCalledWith( defaultProps.domains[ 0 ].name );
-						setPrimaryDomainResolve();
-						setTimeout( () => {
-							expect( component.state( 'settingPrimaryDomain' ) ).toBe( false );
-							expect( component.state( 'changePrimaryDomainModeEnabled' ) ).toBe( false );
-							done();
-						}, 0 );
-					} );
-				} );
-
-				test( 'should handle errors and revert the optimistic updates', () => {
-					return new Promise( ( done ) => {
-						component.instance().handleUpdatePrimaryDomain( 0, defaultProps.domains[ 0 ] );
-						setPrimaryDomainReject( { error: 'Message' } );
-						setTimeout( () => {
-							expect( component.state( 'settingPrimaryDomain' ) ).toBe( false );
-							expect( component.state( 'changePrimaryDomainModeEnabled' ) ).toBe( true );
-							expect( component.state( 'primaryDomainIndex' ) ).toBe( 1 );
-							done();
-						}, 0 );
-					} );
-				} );
-			} );
-		} );
-
-		describe( 'when less than 2 domains', () => {
-			beforeEach( () => {
-				const oneDomain = deepFreeze( {
-					domains: [ { name: 'example.com', isPrimary: true } ],
-					isRequestingSiteDomains: true,
-				} );
-				const propsWithOneDomain = deepFreeze( Object.assign( {}, defaultProps, oneDomain ) );
-				component = renderWithProps( propsWithOneDomain );
-			} );
-
-			test( 'should not show "Change Primary Domain" button', () => {
-				const button = component.find( {
-					className: 'list__change-primary-domain',
-				} );
-				expect( button ).toBeDisabled();
-			} );
-		} );
-	} );
-
```

Comment:
This test set tested the primary domain card and `set primary domain mode` that were removed.